README.adoc

```diff
@@ -11,6 +11,79 @@ This repository follows the monorepo pattern and includes the following projects
 - **verifier**: A command line application to audit ballots and verify the election results.
 - **voting_schemes**: Implementation of different cryptographic voting schemes to be used to run the elections. Currently, it includes the Dummy scheme, used only for testing purposes, and the ElectionGuard scheme, based on the Python implementation of the protocol.
 
-## License
+== Getting started with this repository
+
+- Clone this repository with submodules `git clone --recurse-submodules -j8 https://github.com/decidim/decidim-bulletin-board.git`
+- Or clone the submodules separately after the initial clone by running `git submodule update --init --recursive` in the repository folder
+- Use one of the development methods below
+
+=== Devcontainer
+
+This repository ships with a dev container to get you up and running quickly. This make is easy to get the development environment running without having to install all the dependencies.
+
+Read more about https://code.visualstudio.com/docs/remote/containers[devcontainers on Visual Studio Code documentation].
+
+=== Manual development environment
+
```

Comment:
```suggestion

These instructions assume you're using the last stable version of Ubuntu (Ubuntu 22.04 LTS). It should work for other GNU/Linux based systems, but you'll need to translate the commands to your environment. For instance, instead of using apt, using your distribution package manager, the names of the packages will also change, etc. 

```